model_cards/Geotrend/bert-base-ar-cased/README.md

```diff
@@ -0,0 +1,46 @@
+---
+tags:
+- bert
+- ar
+- masked-lm
+- pytorch
+- tf
```

Comment:
```suggestion
language: ar
```

This should be a language tag, and the other tags are automatic

Reply:
Corrected in https://github.com/huggingface/transformers/pull/8617/commits/cf75587e11f76b5a3d1dcd90fb7c612704dd3d70
Thanks for the review @julien-c